services/device-registry-base/src/main/java/org/eclipse/hono/service/tenant/TenantService.java

```diff
@@ -86,9 +87,9 @@
      *             <ul>
      *             <li><em>200 OK</em> if a tenant with a matching trusted certificate authority exists.
      *             The <em>payload</em> will contain the tenant's configuration information.</li>
+     *             <li><em>400 Bad Request</em> if the certificate authority is missing.</li>
```

Comment:
This is a Java interface defining the behavior of implementations. The existence of required parameters passed in via a remoting protocol is the responsibility of the HTTP and/or AMQP endpoints that sit in front of the Java based implementation of this service interface. Thus it makes perfect sense that the methods of this interface require the passed in parameters to not be null. Please revert all corresponding changes.

Reply:
Okay, I had derived the response from the [FileBasedTenantService#L320-L333](https://github.com/bosch-io/hono/blob/17179adb6933c0eb3d1eeeccc26518e27297d977/services/device-registry-file/src/main/java/org/eclipse/hono/deviceregistry/file/FileBasedTenantService.java#L320-L333).
Thanks for clarifying.